ecosystem/sep-0031.md

```diff
@@ -68,6 +68,30 @@ All endpoints respond with content type:
 
 - `application/json`
 
+## Asset Conversions and Liquidity Strategies
+
+This protocol can be used if the Sending and Receiving Client wish to send and receive the same asset, but this is typically not the case. It is more common for the Sending Client to provide the Sending Anchor with a particular asset (the source asset) and have the Receiving Client receive a different asset (the destination asset).
+
+In this case, the source asset must convert into the receiving asset at some point prior to delivering funds to the Receiving Client. This conversion must be done by either the Sending Anchor, the Stellar Network, or the Receiving Anchor. The subsections below outline the stratgies used for each method.
+
+### Sending Anchor Asset Conversion
+
+The Sending Anchor can collect the source asset from the Sending Client and send the destination asset to the Receiving Anchor. 
+
+This approach requires the Sending Anchor to provide an exchange rate, or quote, to the Sending Client. It also requies the Sending Anchor to hold a balance of the destination asset on Stellar. The Sending Anchor would use Payment operations to send the quoted amount of the destination asset to the Receiving Anchor.
+
+### Stellar Network Asset Conversion
+
+Instead of holding a balance of the destination asset on Stellar, the Sending Anchor can instead us Path Payments to convert the source asset to the destination asset on the Stellar Network. 
+
+This approach requires a sufficiently liquid market to exist between the source and destination assets on the Stellar Decentralized Exchange (SDEX). Note that the exchange rate between the two assets is dynamic due to market fluctuations on the SDEX.
```

Comment:
>Note that the exchange rate between the two assets is dynamic due to market fluctuations on the SDEX.

❓ This is mentioned here as if it is unique to the SDEX, but in all cases the rate between two assets will be fluid. What's the intention with calling it out for only the SDEX?

Reply:
Fair point. I can remove it 👍